components/authz-service/storage/postgres/migration/sql/49_update_rule_triggers.up.sql

```diff
@@ -0,0 +1,50 @@
+
+BEGIN;
+
+CREATE OR REPLACE FUNCTION update_rule(in_rule_id text, in_project_id text, in_name text, in_type text, in_deleted boolean, projects_filter TEXT[])
```

Comment:
[super nit] It's a good practice in sql functions to start the variable names with an underscore (e.g. `_in_rule_id text`) to avoid potential future conflict with tables, columns, etc with the same name that might get added in the future. Probably pretty safe in this case with the `in_` prefixes though.